develop.sh

```diff
@@ -13,6 +13,7 @@ function invoke_docker_compose_spark {
 
 function invoke_docker_compose {
 	docker-compose -f docker/docker-compose.yml \
+	 			-f docker/docker-compose.spark.yml \
```

Comment:
I'm not sure what specifying two `-f` params does exactly. What happens if we run `./develop.sh up`? Does it bring up both the spark containers and the webserver containers?

Reply:
Mentioning multiple `-f` combines the `.yml` files in the order they are mentioned and then brings them up. Yes `./develop.sh up` brings both spark and webserver containers up